job_controller/job_controller.go

```diff
@@ -3,7 +3,13 @@ package job_controller
 import (
 	"errors"
 	"fmt"
+	commonv1 "github.com/kubeflow/common/operator/v1"
```

Comment:
Could we place third-party deps after the first group? Ref https://github.com/golang/go/wiki/CodeReviewComments#imports